mne/tests/test_epochs.py

```diff
@@ -1826,7 +1844,7 @@ def test_array_epochs():
     assert_allclose(epochs.times, [0.])
     assert_allclose(epochs.get_data(), data[:, :, :1])
     epochs.save(temp_fname)
-    epochs_read = read_epochs(temp_fname)
+    epochs_read = read_epochs(temp_fname, add_eeg_ref=False)
```

Comment:
Confirmed that setting `read_epochs(..., avg_eeg_ref=False)` allows these `EpochsArray` changes to be reverted without breaking any other tests.